pywinauto/unittests/test_HwndWrapper.py

```diff
@@ -694,6 +695,18 @@ def testDragMouseInput(self):
         self.ctrl.drag_mouse_input("left", birds.rectangle().mid_point(), dogs.rectangle().mid_point())
         dogs = self.ctrl.GetItem(r'\Dogs')
         self.assertEquals([child.Text() for child in dogs.children()], [u'Birds', u'Dalmatian', u'German Shepherd', u'Great Dane'])
+		
+    def testVerifyEnabled(self):
+        "test for verify_enabled"
+        self.dlg.TabControl.Select(4)
+        editBox = self.dlg.EditBox.WrapperObject()
+        self.assertRaises(ElementNotEnabled, editBox.verify_enabled)
+
+    def testVerifyVisible(self):
+        "test for verify_visible"
+        editBox = self.dlg.EditBox.WrapperObject()
```

Comment:
@a-zotov, it's up to you. Just choose more logically appropriate class for the tests. Asking a lot of questions wastes our time that is almost over.